pkg/proxy/util/utils.go

```diff
@@ -56,3 +72,54 @@ func ShouldSkipService(svcName types.NamespacedName, service *api.Service) bool
 	}
 	return false
 }
+
+// GetNodeAddresss list all match node IP addresses based on given cidr list.  Inject NetworkInterface for test purpose.
```

Comment:
Comment and code do not match.

Should be named `GetNodeAddresses()`

Reply:
Nice finding, it's fixed now. Thanks!